### ISSUE TITLE
Can't install with nokogiri 1.11.0.rc3

### ISSUE THREAD

**Issue by casperisfine** [rubys/nokogumbo#149]
Nokogiri is working on shipping with precompiled binaries, if one of these precompiled nokogiri is installed it cause nokogumbo to fail:

```
 extconf.rb:67:in `join': no implicit conversion of nil into String (TypeError)
        from extconf.rb:67:in `<main>'
extconf failed, exit code 1
```

**Comment by flavorjones** [comment]
@casperisfine - thanks for opening this. This is a known issue - https://github.com/sparklemotion/nokogiri/pull/1788 - should be fixed for the next nokogiri v1.11.0 release candidate.

(I think this issue should be closed, because I think the bug is with Nokogiri breaking an implicit contract with Nokogumbo.)

**Comment by casperisfine** [comment]
👌
